cython.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: cython
-  version: 0.29.36
+  version: 3.0.0
```

Comment:
Let's check this - it seems a bit of a leap

Reply:
Yeah but only to verify that it doesn't break anything python-wise

Reply:
So I checked and below are my findings, below packages were dependent of cython:

:white_check_mark: cqlsh
:white_check_mark: libseccomp
:x:  py3-yaml

The `py3-yaml` failed with error:

```
⚠️  aarch64   |     ext.sources = self.cython_sources(ext.sources, ext)
⚠️  aarch64   |                   ^^^^^^^^^^^^^^^^^^^
⚠️  aarch64   |   File "/usr/lib/python3.11/site-packages/setuptools/_distutils/cmd.py", line 107, in __getattr__
⚠️  aarch64   |     raise AttributeError(attr)
```

On looking it further it seems to be an open issue and the only way I could find was [this](https://github.com/yaml/pyyaml/issues/601#issuecomment-1660456820) which involves using `pip` which I suppose we shouldn't use so this might remain broken if we don't figure out some other way or use pip